partcad-cli/src/partcad_cli/click/commands/list/assemblies.py

```
@@ -0,0 +1,76 @@
+)
+@click.option(
+    "-u",
+    "--used_by",
+    type=str,
+    required=False,
+    help="Only process objects used by the given assembly or scene.",
+    show_envvar=True,
+)
+@click.argument("package", type=str, required=False, default=".")  # help="Package to retrieve the object from"
+@click.pass_obj
+def cli(ctx, recursive, used_by, package):
```

Ruff (0.8.2)

`cli` is too complex (11 > 10)

(C901)